### PR TITLE
Feature/update policy

### PR DESCRIPTION
**Pull request by Marianasilvago**
### What

Add a update policy endpoint

### How to review

Review the code changes.

### Who can review

Anyone.

### CODE REVIEW

File: models/policies.go [hunk 1]
```diff
@@ -28,23 +36,30 @@ type Policy struct {
 	Conditions []Condition `bson:"conditions" json:"conditions,omitempty"`
 }
 
-type NewPolicy struct {
+type UpdateResult struct {
+	ModifiedCount int
+	UpsertedCount int
+}
+
+type PolicyInfo struct {
```
Comment (daiLlew) [inline models/policies.go:44]: Same here.

File: models/policies.go [hunk 2]
```diff
@@ -28,23 +36,30 @@ type Policy struct {
 	Conditions []Condition `bson:"conditions" json:"conditions,omitempty"`
 }
 
-type NewPolicy struct {
+type UpdateResult struct {
+	ModifiedCount int
+	UpsertedCount int
```
Comment (daiLlew) [inline models/policies.go:41]: Should this model also have JSON tags?

Comment (Marianasilvago) [reply models/policies.go:41]: Its an internal struct used to return results from repository layer to api layer, so we dont need a JSON tag I think.

**Comment by pidah** [comment]
LGTM 👍🏼

**Review by daiLlew — APPROVED** [review]

**Pull request merged**